### PR TITLE
Adding label schema

### PR DESCRIPTION
**Pull request by waja**
See more at http://label-schema.org/ and http://label-schema.org/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,23 @@ FROM php:7.2-fpm-alpine
...
 MAINTAINER Thomas Trautmann <thomas.trautmann@tmt.de>
 
+ARG BUILD_DATE
+ARG BUILD_VERSION
+ARG VCS_URL
+ARG VCS_REF
+ARG VCS_BRANCH
+
+# See http://label-schema.org/rc1/ and https://microbadger.com/labels
+LABEL org.label-schema.name="PHP 7.1 - FastCGI Process Manager" \
```
Comment (nItroFreeZer) [inline Dockerfile]: Wrong version number in name label.

File: Dockerfile [hunk 2]
```diff
@@ -2,6 +2,23 @@ FROM php:7.2-fpm-alpine
...
+    org.label-schema.description="PHP-FPM 7.1 (with some more extentions installed)" \
```
Comment (nItroFreeZer) [inline Dockerfile]: Wrong version number in description label.

**Review by nItroFreeZer — CHANGES_REQUESTED** [review]
Wrong version numbers in label values.

**Comment by waja** [comment]
Please check out the new pushed commit

**Review by nItroFreeZer — APPROVED** [review]

**Pull request merged**